src/inverted_index/inverted_index.c

```diff
@@ -932,6 +942,16 @@ bool read_flags_wide(IndexBlockReader *blockReader, const IndexDecoderCtx *ctx,
   return readFlagsWide(blockReader, ctx, res);
 }
 
+// Wrapper around the private static `readFlagsOffsets` function to expose it to benchmarking.
+bool read_flags_offsets(IndexBlockReader *blockReader, const IndexDecoderCtx *ctx, RSIndexResult *res) {
```

Comment:
Consider fixing the name, at least for the wrapper

Reply:
Exactly
The encoder is called `encodeFieldsOffsets` but the decoder `readFlagsOffsets`. Seems like a typo and that they both should say “fields”

Reply:
Oh right. That confused me as well at first and then I got used to it. :)

Renamed.